deploy/examples/csi/cephfs/snapshotclass.yaml

```diff
@@ -3,7 +3,10 @@ apiVersion: snapshot.storage.k8s.io/v1
 kind: VolumeSnapshotClass
 metadata:
   name: csi-cephfsplugin-snapclass
-driver: rook-ceph.cephfs.csi.ceph.com # driver:namespace:operator
+# The prefix of the driver name can be configured by updating
+# CSI_DRIVER_NAME_PREFIX in the rook-ceph-operator-config configmap by default
+# it is namespace name where rook operator is running i.e rook-ceph.
+driver: .cephfs.csi.ceph.com # csi-provisioner-name
```

Comment:
We still need to keep the default `rook-ceph` prefix there. And actually I'm thinking we don't need lines 6-8 documented in every storage class. It's not going to be a common scenario to update the storage class, so they can just use the documentation topic, and no need to document here. If we just have the `# csi-provisioner-name` tag, then they can easily search where to update it.